tests/test_snapshot.py

```diff
@@ -425,3 +426,193 @@ async def test_snapshot_tracestats(agent, tmp_path, snapshot_ci_mode, do_referen
 
         resp = await agent.get("/test/session/snapshot", params={"test_session_token": "test_case"})
         assert resp.status == 200, await resp.text()
+
```

Comment:
can we add one more test case ensuring that when `span_id` is set as a removed attribute that the request fails?